lib/services/local/access_list.go

```diff
@@ -167,37 +173,53 @@ func (a *AccessListService) UpdateAccessList(ctx context.Context, accessList *ac
 
 type opFunc func(context.Context, *accesslist.AccessList) (*accesslist.AccessList, error)
 
+func validateOwnerList(acl *accesslist.AccessList) error {
+	ownerMap := make(map[string]struct{}, len(acl.Spec.Owners))
+	for _, owner := range acl.Spec.Owners {
+		if _, ok := ownerMap[owner.Name]; ok {
+			return trace.AlreadyExists("owner %s already exists in the owner list", owner.Name)
+		}
+		ownerMap[owner.Name] = struct{}{}
+	}
+	return nil
+}
```

Comment:
Why we need a separation function for this ? 

Does it sounds reasonable to validate owners  within  the accessList.CheckAndSetDefaults  function ?